src/platform/forms-system/src/js/types.js

```diff
@@ -207,46 +207,47 @@
 
 /**
  * @typedef {Object} UIOptions
- * @property {string} [ariaDescribedby] - The id of the element that describes the field. Use `messageAriaDescribedby` for web components.
- * @property {string} [classNames] - additional CSS classes to add to the field
- * @property {boolean} [confirmRemove] - For arrays. If true, will show a confirmation modal when removing an item.
- * @property {string} [confirmRemoveDescription] - For arrays. Description for the confirmation modal when removing an item.
- * @property {string} [customTitle] - For the review page, for arrays and some widgets. This doesn't appear to change any text, but is just used for a hack to prevent an outer DL wrapper. Often set to `' '`, and used with `useDlWrap: true` to get a11y issues to pass. Will format field title and body vertically instead of horizontally. `useDlWrap` will format text horizontally.
- * @property {number} [debounceRate] - Used for AutoSuggest widget
- * @property {number} [doNotScroll] - For arrays. By default when adding a new item it will scroll to the next item. Set this to true to disable that behavior.
- * @property {string} [duplicateKey] - For arrays.
- * @property {boolean} [enableAnalytics] - Enable google analytic events. Sent on blur. Use a browser extension such as Adswerve to view the events in the console.
- * @property {string} [expandUnder] - The key of the uiSchema directly before this field
- * @property {boolean} [expandContentFocus] - Used with expandUnder. When the field expands under, it exclusively shows a vertical, blue bar, is indented, and focuses on the field's input.
+ * @property {string} [ariaDescribedby] The id of the element that describes the field. Use `messageAriaDescribedby` for web components.
+ * @property {string} [classNames] additional CSS classes to add to the field
+ * @property {boolean} [confirmRemove] For arrays. If true, will show a confirmation modal when removing an item.
+ * @property {string} [confirmRemoveDescription] For arrays. Description for the confirmation modal when removing an item.
+ * @property {string} [customTitle] For the review page, for arrays and some widgets. This doesn't appear to change any text, but is just used for a hack to prevent an outer DL wrapper. Often set to `' '`, and used with `useDlWrap: true` to get a11y issues to pass. Will format field title and body vertically instead of horizontally. `useDlWrap` will format text horizontally.
+ * @property {boolean} [currency] For `'ui:webComponentField': VaNumberInputField`
```

Comment:
This is the only new line in this file. 
From the rest of the file, removed "-" from most lines because that adds an unnecessary bullet point to the hover info